staging/src/k8s.io/apimachinery/pkg/util/httpstream/wsstream/conn.go

```diff
@@ -256,6 +268,21 @@ func (conn *Conn) handle(ws *websocket.Conn) {
 		if len(data) == 0 {
 			continue
 		}
+		if supportsStreamClose && data[0] == remotecommand.StreamClose {
+			if len(data) != 2 {
+				klog.Errorf("Single channel byte should follow stream close signal. Got %d bytes", len(data)-1)
+				break
+			} else {
+				channel := data[1]
```

Comment:
do we need to do the same transform as below?

```
		if conn.codec == base64Codec {
			channel = channel - '0'
		}
```

Reply:
A `base64` protocol is defined, but I don't think any clients are using it. Definitely not the new `WebSocketExecutor` client. I can add it anyways, since it shouldn't affect anything.